.teamcity/_self/projects/WPComTests.kt

```diff
@@ -297,7 +297,7 @@ fun gutenbergPlaywrightBuildType( viewportName: String, buildUuid: String ): Bui
                     export NODE_CONFIG="{\"calypsoBaseURL\":\"${'$'}{URL%/}\"}"
                     export DEBUG=pw:api
 
-                    xvfb-run yarn jest --reporters=jest-teamcity --reporters=default --testNamePattern @parallel --maxWorkers=%E2E_WORKERS% specs/specs-playwright-wpcom
+                    xvfb-run yarn jest --reporters=jest-teamcity --reporters=default --maxWorkers=%E2E_WORKERS% --config jest.config.gutenberg.js
```

Comment:
I think we had tests in `specs/specs-playwright-wpcom` that were testing ETK, not Gutenberg. But not sure.

Reply:
Would ETK be better served being split into its own build config?

Reply:
I wonder if it makes sense to frame these tests as targeting the wpcom editor generally rather than just gutenberg or ETK? The reality is that we are testing pretty much all code served for the editor just by loading it, which includes ETK and gutenberg